src/BankAccountSecondary.java

```diff
@@ -0,0 +1,78 @@
+import components.map.Map;
+
+public abstract class BankAccountSecondary implements BankAccount {
+    /**
+     * Total Accounts or Size of Map
+     */
+    int size;
+
+    /**
+     * Map with all user accounts
+     */
+    Map<String, Account> accounts;
```

Comment:
There should be no fields at all here. You should be able to access everything you need via kernel methods.